app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style: You should add type hinting to your function parameters and return values. It's a good practice to make your code more readable and maintainable. In this case, you should specify the type of `cafe` parameter. It's clear from the import statement that it should be of `Cafe` type.